core/build.gradle.kts

```
@@ -397,9 +397,11 @@ tasks.withType<AbstractDokkaLeafTask>().configureEach {
     dokkaSourceSets.configureEach {
         // reportUndocumented.set(true) // much noisy output about `hashCode` and serializer encoders, decoders etc
         skipDeprecated.set(true)
+        // Enum members and undocumented toString()
+        suppressInheritedMembers.set(true)
         // hide the `internal` package, which, on JS, has public members generated by Dukat that would get mentioned
         perPackageOption {
-            matchingRegex.set(".*\\.internal\\..*")
+            matchingRegex.set(".*\\.internal(\\..*|)")
```

```suggestion
            matchingRegex.set(".*\\.internal(\\..*)?")
```
A bit more readable, no?